kuma/settings/common.py

```diff
@@ -438,6 +438,7 @@ def enable_candidate_languages():
     'kuma.core.context_processors.next_url',
 
     'constance.context_processors.config',
+    'debreach.context_processors.csrf',
```

Comment:
From reading the source code, I think this actually replaces `django.core.context_processors.csrf`, but I decided to follow instructions and just tack it at the end.